_datascience/deidentification.md

```diff
@@ -1,114 +1,173 @@
-randomization of coded identifiers. For data that may need to be re-identified
-later, retaining appropriate documentation of the mapping between identifiable
-data from patients, specimens or datasets, and the coded identifiers generated
-is critical. Depending on the study design and protocol, the entity retaining
-this mapping documentation will be responsible for preventing non-approved
-re-identification. In some cases if the research group maintains this mapping
-documentation then their research will still be considered human subjects,
-though if the mapping is maintained by a non-involved 3rd party the research may
-be considered exempt.
-
-There is often a question of whether the HIPAA de-identification process is
-enough to render de-identified specimens/private information sufficiently
-non-identifiable for exemption from human subjects research. A specimen
-de-identified under HIPAA is not the same as a non-identifiable specimen for
-research (Office of Human Research Protection). Under research, a specimen
-rendered non-identifiable under 45 CFR 46.102(f) could qualify as exempt from
-human subjects research. A non-identifiable specimen (thus not human subject
-research) must meet the following two requirements:
-
-1.  The private information or specimens are NOT collected for a specific
-research project through an interaction or intervention with living
-individuals; and
-
-2.  The investigator(s) cannot readily ascertain the identity of the
-individual(s) to whom the coded private information or specimens pertain
-because, for example:
-
--   The investigators and the holder of the key enter into an agreement
-    prohibiting the release of the key to the investigators under any
-    circumstances, until the individuals are deceased (note that the HHS
-    regulations do not require the IRB to review and approve this agreement);
-
--   There are IRB-approved written policies and operating procedures for a
-    repository or data management center that prohibit the release of the key
-    to the investigators under any circumstances, until the individuals are
-    deceased; or
-
--   There are other legal requirements prohibiting the release of the key to the investigators, until the individuals are deceased.
-
-## How can a genomic dataset be de-identified?
-
-If all direct and indirect identifiers are removed from a genomic data set, the
-genomic data itself may still be able to identify a patient depending on the
-data type and degree of processing of the specific data entity. The inclusion of
-racial, ethnic and genders in scientific research however, can be a reason to
-retain some of these indirect identifiers in the context of research datasets.
-Thus, in addition, confidentiality is supported through strong use limitations,
-data use agreements and appropriate data security.
+There are two important federal regulations related to the use of "individually
+identifiable health information." The first, the [HIPAA Prvacy
```

Comment:
This is great @monicagerber! Adding a few minor suggestions: it might help readability to break up the text in cases like this and show a numbered list

Reply:
Adjusted!